Added hmac.h to Io header in addon SHA1 to fix implicit decleration.
https://github.com/IoLanguage/io/pull/359
https://github.com/IoLanguage/io/commit/e707943b623c9bf680ac6a5bd54646ce1b5a61bd
--- addons/SHA1/source/IoSHA1.h.orig
+++ addons/SHA1/source/IoSHA1.h
@@ -8,6 +8,7 @@
 #include "IoObject.h"
 #include "IoNumber.h"
 #include "sha1.h"
+#include "hmac.h"
 
 #define ISSHA1(self) IoObject_hasCloneFunc_(self, (IoTagCloneFunc *)IoSHA1_rawClone)
 
